base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -264,6 +264,9 @@ impl LMDBDatabase {
                 UpdateDeletedBlockAccumulatedDataWithDiff { header_hash, deleted } => {
                     self.update_deleted_block_accumulated_data_with_diff(&write_txn, header_hash, deleted)?;
                 },
+                UpdateDeletedBitmap { deleted } => {
```

Comment:
I don't like exposing this operation on its own. The deleted bitmap should only ever be updated as part of another operation, so that the database is alway consistent. Where is this operation called on it's own?

Reply:
Agree with that, but trying to avoid refactoring existing code too much. It's used in pruned mode where there are a number of operations that must be atomic (and could potentially be brought together into one operation) to not break the database.

Reply:
Let's add a comment there to discourage it's use